tests/test_project.py

```diff
@@ -665,16 +664,13 @@ def test_job_move(self):
         assert job in project_b
         assert job not in project_a
         assert job == job_b
-        assert hash(job) == hash(job_b)
```

Comment:
Why does this test fail now? Shouldn't it pass after `job` and `job_b` have the same working directory?

Reply:
You're correct. Some of these hashes must still be equal, if  `job == job_b`. Initially I removed this but there isn't a good reason to remove it. I will revert my change and add this back as a test of our adherence to the data model requirement that `job == job_b` implies `hash(job) == hash(job_b)`.